artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/broker/AMQPLargeMessage.java

```diff
@@ -130,6 +130,10 @@ private AMQPLargeMessage(final AMQPLargeMessage copy,
                                   final SequentialFile fileCopy,
                                   final long newID) {
       super(copy);
+      if (this.messageAnnotations != null) {
+         this.messageAnnotations.getValue().forEach((a, b) -> logger.info("{}={}", a, b));
```

Comment:
Logging every annotation on every copy seems excessive especially at INFO, should this be lowered to TRACE to avoid spamming the logs ?